release.nix

```diff
@@ -1,8 +1,16 @@
-{
-  pkgs ? import ./pkgs.nix
-}:
+{ pkgs ? import ./pkgs.nix }:
 
 with pkgs;
 rec {
   application = callPackage ./default.nix {};
+  docker = dockerTools.buildImage {
+    name = application.name;
+    contents = [ application ];
+    extraCommands = ''
+      mkdir -m 1777 tmp
+    '';
+    config = {
+      Cmd = [ "/bin/polykey" ];
```

Comment:
What would the default command for js-polykey be if it were wrapped in the container?

In one case, you would want to run the agent right? Most of the time running in the container means that you're not running interactively.

So I suspect this should instead be `/bin/pk-agent` instead, since you're starting the agent to run autonomously.

But of course docker builds don't work yet. We'll need that working in typescript-demo-lib first before we can finish integration with Matrix agent.